AUTHORS.md

```diff
@@ -0,0 +1,4 @@
+# Authors
```

Comment:
This file is less common nowadays due to the wide spread use of VCSes that commit and tag code contributions from people automatically. Thus you can easily see who wrote what as long as someone clones or uses code templates. I’m just worried having this file might discourage use of the starter kit if people see names they don’t recognize and wonder if they can use code here or need to check for permission.

What are your thoughts?